bin/faucet/src/api/get_tokens.rs

```diff
@@ -238,8 +238,15 @@ impl RawMintRequest {
         // Validate Challenge and nonce
         let challenge_str = self.challenge.ok_or(MintRequestError::MissingPowParameters)?;
         let nonce = self.nonce.ok_or(MintRequestError::MissingPowParameters)?;
-
-        server.submit_challenge(&challenge_str, nonce, account_id, api_key.unwrap_or_default())?;
+        let request_complexity = (asset_amount.base_units() / server.metadata.base_amount) + 1;
```

Comment:
I think there are a couple of places where we compute request complexity based on the amount. It may be good to have a small helper function for this so that in case the formula changes, we have just one place to update.